packages/kubernetes/data_stream/state_cronjob/manifest.yml

```
@@ -69,5 +69,5 @@ streams:
         description: >
           Processors are used to reduce the number of fields in the exported event or to enhance the event with metadata. This executes in the agent before the events are shipped. See [Processors](https://www.elastic.co/guide/en/beats/filebeat/current/filtering-and-enhancing-data.html) for details.
 
-    title: Kubernetes Cronjob metrics
+    title: Kube-state-metrics Cronjob metrics
     description: Collect Kubernetes Cronjob metrics from kube_state_metrics
```

```suggestion
    description: Collect Kubernetes Cronjob metrics from kube-state-metrics
```
just for follow the same format as everywhere else